lab_1_classify_by_unigrams/main.py

```diff
@@ -4,33 +4,43 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
-
-
-def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
-    """
-    Calculates frequencies of given tokens
-    :param tokens: a list of tokens
-    :return: a dictionary with frequencies
-    """
-
-
-def create_language_profile(language: str, text: str) -> dict[str, str | dict[str, float]] | None:
-    """
-    Creates a language profile
-    :param language: a language
-    :param text: a text
-    :return: a dictionary with two keys – name, freq
-    """
-
-
-def calculate_mse(predicted: list, actual: list) -> float | None:
+def tokenize(text: str) -> list[str] or None:
```

Comment:
You must not change interface of functions and dock strings - please, revert!

Reply:
done